manifests/prometheus/alerts.d/uaa-5xx-errors.yml

```diff
@@ -0,0 +1,15 @@
+# Source: firehose-exporter
+---
+
+- type: replace
+  path: /instance_groups/name=prometheus2/jobs/name=prometheus2/properties/prometheus/custom_rules?/-
+  value:
+    name: UAA5xxErrorCount
+    rules:
+      - alert: UAA5xxErrorCount
+        expr: sum(increase(firehose_value_metric_uaa_requests_global_status_5_xx_count[6h])) > 10
```

Comment:
I think that's a sufficiently complex expression that it probably warrants a test

Reply:
Decided to add it myself so we can get it out